src/api/web/index.js

```diff
@@ -32,9 +32,8 @@ module.exports = function(config, auth, storage) {
     });
   });
 
-  router.get('/-/logo', function(req, res) {
-    res.sendFile(_.get(config, 'web.logo') || `${env.APP_ROOT}/static/logo-sm.png`
-    );
+  router.get('/-/verdaccio/logo', function(req, res) {
```

Comment:
Why the `/-/verdaccio/` ?

Reply:
We have set 
https://github.com/verdaccio/verdaccio/blob/9cde5adfd4546c7c96cb83f108861b89e41e92ef/src/webui/template/index.html#L9

So every request from web UI is starting with it.

Reply:
That's true, I didn't see that.